data-model/fdo-profiles/0.1.0/annotation/examples/example-annotation-request.json

```diff
@@ -0,0 +1,14 @@
+{
+  "data": {
+    "type": "\"https://hdl.handle.net/21.T11148/cf458ca9ee1d44a5608f",
+    "attributes": {
+      "pidIssuer":"https://ror.org/04wxnsj81",
+      "issuedForAgent":"https://ror.org/0566bfb96",
```

Comment:
Small thing but normally there is a space behind the `:` An autoformat should fix this